docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -52,17 +52,52 @@ An object that defines any variables your tests require.
 [[synthetics-configuration-playwright-options]]
 = `playwrightOptions`
 
-For available options, see the https://playwright.dev/docs/test-configuration[Playwright documentation].
+For all available options, refer to the https://playwright.dev/docs/test-configuration[Playwright documentation]. 
+
+Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including timeouts, timezones, and device emulation.
+
+[discrete]
+[[synthetics-configuration-playwright-options-timeouts]]
+== Timeouts
 
-[NOTE]
-====
 Playwright has two types of timeouts that are used in Elastic Synthetics:
 https://playwright.dev/docs/test-timeouts#action-and-navigation-timeouts[action and navigation timeouts].
 
 Elastic Synthetics uses a default action and navigation timeout of 50 seconds.
 You can override this default using https://playwright.dev/docs/api/class-testoptions#test-options-action-timeout[`actionTimeout`] and https://playwright.dev/docs/api/class-testoptions#test-options-navigation-timeout[`navigationTimeout`]
 in `playwrightOptions`.
-====
+
+[discrete]
+[[synthetics-configuration-playwright-options-timezones]]
+== Timezones
+
+By default monitors will run in the ??? timezone, but you can specify a timezone in
+`playwrightOptions` on a per monitor or global basis.
```

Comment:
What about something like this?

```suggestion
By default, Elastic Synthetics does not set a timezone for synthetic monitors.
Many websites don't require a site visitor's timezone to function, but if any journey in
your project is set up to visit a website that _does_ require a timezone, you can specify
a timezone to use when the monitor runs using `playwrightOptions` on a per monitor
or global basis.
```

Reply:
To be picky, the service doesn't set the timezone but a private location may well have this set.  Also, we are intending to add the timezone to the service with https://github.com/elastic/synthetics-service/issues/1312 in the short term (although I can't say when exactly)

Reply:
Just adding a suggestion. We could start like
```
By default monitors uses the time zone of the host machine. This is not always desirable if we want to test how a given web application behaves across different timezones. 
```